inline.go

```diff
@@ -6,6 +6,7 @@ import (
 	"bytes"
 	"regexp"
 	"strconv"
+	"strings"
```

Comment:
This is not a cleanup. Since the code-blocks now get inline they must be handled by the codeSpan code and need to detect the language.